.github/actions/ui-update/action.yml

```diff
@@ -0,0 +1,93 @@
+name: Update UI
+description: Updates the UI module version in go.mod and creates a PR if there is a new version
+inputs:
+  moduleName:
+    description: 'Module name to update (e.g., walletd)'
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
```

Comment:
I think checkout is on v4 and setup go is on v5. Lets make sure all of these are up to date